maestro-client/src/main/java/maestro/ios/GetRunningAppIdResolver.kt

```diff
@@ -0,0 +1,60 @@
+package maestro.ios
+
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import maestro.debuglog.DebugLogStore
+import okhttp3.HttpUrl
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import java.util.concurrent.TimeUnit
+
+class GetRunningAppIdResolver {
+    private val okHttpClient by lazy {
+        OkHttpClient.Builder()
+            .connectTimeout(10, TimeUnit.SECONDS)
+            .readTimeout(30, TimeUnit.SECONDS)
+            .build()
+    }
```

Comment:
Can you unify okhttp client for the iOS module lets make it a singleton which can be used for this and view hierarchy calls both?